DESCRIPTION

```diff
@@ -12,8 +12,16 @@ Authors@R: c(
 Description: Construct continuous and non-contiguous area cartograms.
 URL: https://github.com/sjewo/cartogram, https://sjewo.github.io/cartogram/
 BugReports: https://github.com/sjewo/cartogram/issues
-Imports: methods, sf, packcircles
-Suggests: 
+Imports: 
```

Comment:
Alternatively, the furrr, parallelly, and future packages could be moved to Suggests: and the code related to them may be only used if the progress bar is on. 

(I also think there should be an option to disable the progress bar)

Reply:
makes sense!

Reply:
@Nowosad On a second thought, that would imply using the for loop (or purrr::map) by default with no progress bar... (though purrr::map also has progress bar). So kind of defeats the purpose. And why would parallel processing rely on the progress bar setting?

Reply:
Thank you for your suggestions and your work on implementing parallel processing @e-kotov! I haven't had much time to keep an eye on this topic in the past.

I would like to keep the imports as small as possible. 

- Maybe `utils::txtProgressBar()` is sufficient for adding a progress bar without additional imports? 
- I would also prefer to move the parallelly, and future packages to suggest, as @Nowosad suggested, and add an if-clause for `n_cpu > 1` to use futures parallel execution.
- I don't want to be petty, but is 'furrr' absolutely necessary? If the code can only be implemented with the future package, I would prefer that.

Best,
Sebastian

Reply:
@sjewo points taken. I'm not sure excluding `furrr` is practical, as `future` package only provides the framework, and then you need some other package like `furrr` or `future.apply` to do the job. So why not `furrr`?

I will get to it next week. What do you think of the older PR #28 ? Did you have plans to merge it at all?

Reply:
Thanks for your reply, @e-kotov ! 

Regarding `furrr`: i would like to keep the number of dependencies low. `future.apply` has half the number of imported packages. Nevertheless, if `furrr` is more practical in your opinion, that's fine for me!

I have not gotten the PR #28 to work and the “future” approach looks promising to me. So if you are working on a different solution that results in some speed improvements, I would certainly accept a PR.